types/jsonwebtoken/jsonwebtoken-tests.ts

```diff
@@ -10,29 +10,33 @@ import fs = require("fs");
 var token: string;
 var cert: Buffer;
 
+interface ITestObject {
+  'foo': 'bar'
+}
+
 /**
  * jwt.sign
  * https://github.com/auth0/node-jsonwebtoken#usage
  */
 // sign with default (HMAC SHA256)
-token = jwt.sign({ foo: 'bar' }, 'shhhhh');
+token = jwt.sign({ foo: 'bar' } as ITestObject, 'shhhhh');
 
 // sign with default (HMAC SHA256) and single audience
-token = jwt.sign({ foo: 'bar' }, 'shhhhh', { audience: "theAudience"});
+token = jwt.sign({ foo: 'bar' } as ITestObject, 'shhhhh', { audience: "theAudience"});
 
 // sign with default (HMAC SHA256) and multiple audiences
-token = jwt.sign({ foo: 'bar' }, 'shhhhh', { audience: ["audience1", "audience2"]});
+token = jwt.sign({ foo: 'bar' } as ITestObject, 'shhhhh', { audience: ["audience1", "audience2"]});
 
 // sign with default (HMAC SHA256) and a keyid
-token = jwt.sign({ foo: 'bar' }, 'shhhhh', { keyid: "theKeyId"});
+token = jwt.sign({ foo: 'bar' } as ITestObject, 'shhhhh', { keyid: "theKeyId"});
 
 // sign with RSA SHA256
 cert = fs.readFileSync('private.key');  // get private key
-token = jwt.sign({ foo: 'bar' }, cert, { algorithm: 'RS256'});
+token = jwt.sign({ foo: 'bar' } as ITestObject, cert, { algorithm: 'RS256'});
 
 
 // sign asynchronously
-jwt.sign({ foo: 'bar' }, cert, { algorithm: 'RS256' }, function(err: Error, token: string) {
+jwt.sign({ foo: 'bar' } as ITestObject, cert, { algorithm: 'RS256' }, function(err: Error, token: string) {
```

Comment:
foo is not a known property and therefore the type checking fails unless you specify that foo should exist.